API/ev3_lcd.c

```diff
@@ -1880,5 +1880,5 @@ int Ev3Println(const char *fmt, ...)
 void Ev3Clear()
 {
 	LcdClean();
-	LcdPrintf("\f");

```

Comment:
How is that a typo? There is no necessity to use "%s" here.

Reply:
I just checked. The read typo was the missing 1 as first argument. That is why the Ev3Printf is a good idea.
Since I just added the `"%s"` instead of fixing the actual issue, this commit is useless.